.githooks/pre-push

```diff
@@ -2,7 +2,7 @@
 
 echo "Running static analysis with checkstyle and ktlint..."
 
-./gradlew checkstyle ktlint --daemon
+./gradlew checkstyle detekt --daemon
```

Comment:
would this cover the example app as well?
`./gradlew :example:ktlint`

Reply:
actually I would need to modify the detekt file so it is enabled in that module too.  I will make that update.

Reply:
yes